js/app/modules/card/video.js

```diff
@@ -30,6 +30,20 @@ const Video = new Module.Class({
         'show-title':  GObject.ParamSpec.boolean('show-title', 'Show Title Label',
             'Whether to show the title label',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY, true),
+        /**
+         * Property: previous-card
+         * Card linking to the previous document card.
+         */
+        'previous-card': GObject.ParamSpec.object('previous-card',
+            'Previous Card', 'Previous Card',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY, Gtk.Widget),
+        /**
+         * Property: next-card
+         * Card linking to the next document card.
+         */
+        'next-card': GObject.ParamSpec.object('next-card',
```

Comment:
@tchx84 is removing these in favour of a `nav-content` property and his pull request is ready to be merged, so I don't want to merge this as-is right now. Could you rebase it on T16699? Maybe it would be a good idea to move the property to the ArticleContent interface as well.